src/main/settings.py

```diff
@@ -250,3 +251,6 @@
 # STRIPE
 STRIPE_PUBLIC_KEY = os.environ.get("STRIPE_PUBLIC_KEY", "")
 STRIPE_API_KEY = os.environ.get("STRIPE_API_KEY", "")
+STRIPE_WEBHOOK_SECRET = os.environ.get("STRIPE_WEBHOOK_SECRET", "")
+STRIPE_API_VERSION = os.environ.get("STRIPE_API_VERSION", "")
+ESSENTIAL_PLAN_NAME = os.environ.get("ESSENTIAL_PLAN_NAME", "")
```

Comment:
Use a constant, maybe in payment/constants.py, we don't necessarily to have this dynamic, but reusable